update requirements.txt to avoid library conflicts.

ekivemark opened the pull request:
resolve library version conflict between requests-oauthlib and socialauth

ekivemark committed:
update requirements.txt to avoid library conflicts.

vault_env_django_user_id_hex_status : "False"

whytheplatypus reviewed (dismissed):
Make this update through the `requirements.in` file and sync using [`pip-compile`](https://github.com/jazzband/pip-tools#example-usage-for-pip-compile)

ekivemark committed:
Update_requirements
Installed pip-tools with
pip install pip-tools

in Virtual Environment
then ran pip-compile --output-file requirements/requirements.txt requirements/requirements.in

ekivemark committed:
Update_requirements: Fix imports for Python 2.x

ekivemark committed:
Update_requirements: Fix imports for Python 2.x

ekivemark committed:
Update_requirements: Fix imports for Python 2.x

ekivemark commented:
Updated for flake8 syntax fixes

ekivemark committed:
Update_requirements: dealing with python3-openid for python3
and python-openid for python2

ekivemark committed:
Update_requirements: Fix imports for Python 2.x

ekivemark committed:
Fix README.MD to add --insecure option
This is needed to use local static files (for bootstrap etc.)

ekivemark commented:
This PR also supports Paging in the EOB.
Updated README.MD to add --insecure to runserver instruction. This is now needed to support local static files.
